File: WebAppVen/Scripts/app/model/MiWlocinv.js
Ext.define('WEBAPP.model.MiWlocinv', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'WH_NO', type: 'string' },
        { name: 'MMCODE', type: 'string' },
        { name: 'STORE_LOC', type: 'string' },
        { name: 'LOC_NAME', type: 'string' },
        { name: 'INV_QTY', type: 'string' },
        { name: 'CREATE_DATE', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_DATE', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }/*,
        { name: 'MMNAME_E', type: 'string' }*/
    ]
});

File: Schedule/HIS14SUPDETtransfer/Models/MI_WLOCINV.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HIS14SUPDETtransfer
{
    public class MI_WLOCINV
    {
        public string WH_NO { get; set; }
        public string MMCODE { get; set; }
        public string STORE_LOC { get; set; }
        public string INV_QTY { get; set; }
        public string CREATE_DATE { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_DATE { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string LOC_NOTE { get; set; }
        public string WH_NAME { get; set; }     //庫別名稱
        public string MMNAME_C { get; set; }    //中文品名
        public string MMNAME_E { get; set; }    //英文品名
        public string WH_NAME_DISPLAY { get; set; } //庫房代碼(白色顯示用)
        public string WH_NAME_TEXT { get; set; } //庫房代碼(紅色顯示用)
        public string MMCODE_DISPLAY { get; set; }  //院內碼(白色顯示用)
        public string MMCODE_TEXT { get; set; }     //院內碼(紅色顯示用)
        public string STORE_LOC_DISPLAY { get; set; } //儲位代碼別名(白色顯示用)
        public string STORE_LOC_TEXT { get; set; } //儲位代碼別名(紅色顯示用)
        public string LOC_NOTE_TEXT { get; set; }
        public string CHECK_RESULT { get; set; } //檢核結果
        public string IMPORT_RESULT { get; set; } //異動結果


        public string STRDEPID { get; set; }
        public string STRDRUGID { get; set; }
        public string STRSTOREPOS { get; set; }
        public string LNGDRUGAMT { get; set; }
    }
}
